<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_form</class>
 <widget class="QWidget" name="Add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #AFEEEE;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background: #65CCB8;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 18px</string>
   </property>
   <property name="text">
    <string>Добавление преподавателя:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: white;
</string>
        </property>
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Отчество:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color: white;
</string>
        </property>
        <property name="text">
         <string>Предмет:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;</string>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
        <item>
         <property name="text">
          <string>ОИнфб</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ОУИС</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ТПКИС</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ТЭЦ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ФизК</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ОТС</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СПЭ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ТВиМС</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Физика</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Полит</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>УИП</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>МССвИ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ОТФ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ОЦСТ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ЭП</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>БЖЧ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>НСИК</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ОМО</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: white;
</string>
        </property>
        <property name="text">
         <string>Время:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Кабинет:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Дата:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid  #65CCB8;</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximumDate">
         <date>
          <year>2026</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2022</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background: #65CCB8;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>добавить </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Add_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
